CONTRIBUTING.md

```diff
@@ -0,0 +1,183 @@
+# Contributing
+
+Firstly want to give a gigantic thank you for taking the time to contribute to this project. Your contribution is greatly appreciated.
+
+Before contributing, there is a few things you should know if you don't already.
+
+## Submitting a Pull Request
+
+### Dependencies
+
+In order to install these dependencies locally, you will need to simply run
+
+```bash
+yarn install
+```
+
+### Commit Messages
+
+Evangeline follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification. This means that all commit messages must follow a specific format. This is to ensure that the changelog is generated correctly.
+
+To be more specific, evangeline uses [Angular's commit message format](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type). This means that all commit messages must follow the following format:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+#### Examples
+
+```
+feat: add send method to bot class
+```
+
+```
+refactor!: rename send method to sendMessage
+
+BREAKING CHANGE: send method has been renamed to sendMessage
+```
+
+```
+fix(sendMessage): make sendMessage not accept undefined
+```
+
+**Type**:
+
+The `type` field of your message should be **ONE** of the following:
+
+- build: Changes that affect dependencies or build process
+- chore: Miscellaneous changes that don't affect the code or tests (does not include documentation changes, see `docs`)
+- ci: Changes to CI configuration files and scripts
+- docs: Changes the the documentation
+- feat: Adds a new feature
+- fix: Fixes a bug or issue with the code/library
+- perf: Improves performance in the library
+- refactor: Refactors the code. Usually is not a feature or bug fix.
+- style: Changes that do not affect the meaning of the code (added whitespace, formatting, etc)
+- test: Adds or modifies tests
+- deps: Changes to external dependencies (does not include devDependencies, use `dev-deps`)
```

Comment:
prob add `dev-deps` to the list

Reply:
https://github.com/eludris-community/evangeline/pull/9/commits/58bc0acfd2e41e500d49509109d2c163baf3e75a